docs/operational_guide/replication_and_deployment_in_zones.md

```diff
@@ -0,0 +1,57 @@
+# Replication and Deployment in Zones
+
+## Overview
+
+M3DB supports deployment to both a single zone, with rack level isolation, and deployment across multiple zones in a region. You can also deploy M3DB across multiple regions for a global view of data, however you may experience higher ingestion latency and cross region bandwidth costs. You may want to use a replication factor of five for such a multi-region cluster to survive multiple regions suffering an outage, or be able to still upgrade hosts while a region is down.
+
+### Replication
+
+It is highly recommended that you use a replication factor of at least 3 for any M3DB deployment. This is due to the default write and read consistency levels requiring a majority of hosts to be available to complete an operation successfully. For more information see the documentation concerning tuning [availability, consistency and durability](availability_consistency_durability.md).
+
+### Upgrading hosts in a deployment
+
+It should be noted that when a host is deployed and it is preparing to serve reads by bootstrapping, the shards are available for writes but not available for reads.
+
+There is a small window during an upgrade between when the process is stopped and then started again where the node is unavailable for writes too.
+
+## Deployment in a single zone
+
+For deployment in a single zone, we recommend you set the `isolationGroup` to the name of the rack a host resides or another logical unit that separates groups of hosts in your zone.
+
+In this configuration hosts are distributed shards of data that will not be placed more than once in the same rack or logical unit you define, which allows you to lose an entire unit at a time that may contain multiple hosts and still stay available since it is guaranteed to only affect one replica of data.
+
+Here's what a deployment in a single zone looks like, in this example the cluster has 3 shards that are spread over 4 racks:
+
+![Replication Single Zone](replication_single_zone.png)
+
+Typically deployments have many more than 3 shards, but this is a simple example that depicts how losing a single rack still maintains availability of two of the three replicas of each shard.
```

Comment:
maybe: "still maintains availability of the entire cluster for writes and reads because we still have 2 full replication factors for all of the shards which is enough to achieve quorum"